### DISCUSSION TITLE
Smart Watering System with Internet of Things (Project 219)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#51] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 219](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Smart%20Watering%20System%20with%20Internet%20of%20Things)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Shubhangi855** [comment]
"Hi, I am a student at birla institute of technology and interested in working and collaborating on this project".

**Reply by grsasaki-tmw** [reply]
Thank you for your interest in this project!  If you have any questions or need any guidance I encourage you to post in this discussion so that others may benefit in the future.

**Comment by RAJATH1235656** [comment]
Hi, I am a student at rv College of engineering and interested in working and collaborating on this project

**Comment by Sabin22** [comment]
Salut !
Je suis Sabin, étudiant à la faculté polytechnique et désire collaborer pour ce projet.

**Reply by grsasaki-tmw** [reply]
Hello Sabin, thank you for your interest in this project and reaching out to us.  If you would like to collaborate further or have questions, please feel free to post in this Discussion Board.  Thank you!  - Gen Sasaki, MathWorks

**Comment by RAJATH1235656** [comment]
Hi,
I am rajath from RV College of engineering Bangalore, Karnataka, India.
I am at electronics and instrumentation student. My college email ID is
***@***.***

Hi would like to know what kind of support you can provide and what are
your expertise.
I am providing my resume so you can go to my skills.

Thank you.

**Reply by grsasaki-tmw** [reply]
Hello Rajath, thank you for your interest in this project and reaching out to us.  As I replied to another post above, if you would like to collaborate further or have questions, please feel free to post in this Discussion Board.  The type of support we can provide here are mainly to help you with applying the MathWorks tools to the project.  If you have any specific questions about a MATLAB feature I would recommend you go to our [Support Page](https://www.mathworks.com/support.html).  We might also be able to handle general engineering questions, as our team has various engineering backgrounds in industry.  - Gen Sasaki, MathWorks

**Comment by rashamy2002** [comment]
Hi. I am lecturer and lab engineer in red sea University Sudan. I am interested in collaborating in this project.

**Reply by grsasaki-tmw** [reply]
Thank you for your interest in the project!  As mentioned above I am happy to collaborate with you on your project.  Please feel free to post in these discussions with your questions and ideas.  Again, general questions about MATLAB I recommend that you refer to our [Support Page](https://www.mathworks.com/support.html) where you can even contact our excellent Technical Support group in one of the links at the bottom of that page. - Gen Sasaki, MathWorks

**Comment by nyahbm** [comment]
Hi. We are a team of five undergrad seniors at Boston University interested in working on this project for the 2022-23 school year.

**Reply by grsasaki-tmw** [reply]
Thank you for your interest in the project! As written above, please do let me know in this discussion here if you have any questions.

**Comment by RAJATH1235656** [comment]
Thank you for your response. I have started on IOT part of project.
Circuits are in verification process and when Circuits are ready , I will
contact you.  It will be ready in 1 month.

Thanks & Regards

**Comment by saadjawad79** [comment]
Hi! I am an electrical engineering undergraduate and I am interested in collaborating in this project.

**Comment by Naima-Yaqub** [comment]
Hi! I am an electrical engineering student at NUST. I am interested in working on this project.

**Comment by SivagurubalaSwaminathan** [comment]
Hello , I'm an Electronic communication Engineering student at KPRIET. I'm interested in working on this project.

**Comment by kowsalya962** [comment]
Hi, I am Kowsalya from IIT Madras. Looking for people to collaborate with for this project. Thanks!

**Comment by SivagurubalaSwaminathan** [comment]
Sure we shall !

**Reply by kowsalya962** [reply]
Great! Shall we collaborate?

**Comment by SivagurubalaSwaminathan** [comment]
Yes we shall !

**Reply by kowsalya962** [reply]
Great! Kindly give me any contact for discussing about the project.

**Reply by dinwisp** [reply]
Shall I be a part of this?

**Reply by kowsalya962** [reply]
yeah! Kindly email me at ic37440@imail.iitm.ac.in for further discussion of the project.

**Comment by SivagurubalaSwaminathan** [comment]
***@***.***
This is my personal mail id , we shall contact through this .

**Reply by kowsalya962** [reply]
I am sorry. But your email is not visible to me.

**Comment by vk1730** [comment]
Hi, I am a student at SRM IST and am interested in working and collaborating on this project

**Reply by vk1730** [reply]
vkprbhkr663@gmail.com

**Comment by SivagurubalaSwaminathan** [comment]
Can you please share me your email id?

**Comment by Ramana8723** [comment]
Hello , I am Ramana from Sri Venkateswara college of engineering, Chennai. I am interested in working on this project with my junior.

**Reply by dinwisp** [reply]
yep!

**Comment by neharika950** [comment]
hello,
Should i be doing the advanced project works to receive the rewards mentioned?

**Reply by robertogl** [reply]
No that is not necessary.

**Comment by Soutehkeshan** [comment]
Hi, My name is Arad and I'm a second year Computer Science Student at the University of Greenwich with a focus on AI and I'm interested in working on this project.

**Reply by dinwisp** [reply]
Count me in

**Reply by srikar-burugula** [reply]
HI @Gonzaleski!
I am interested in collab for this Project

**Reply by Soutehkeshan** [reply]
Hi @srikar-burugula,
Sorry, I have already submitted my project for 2024.
Good luck!
